charts/headlamp/values.yaml

```diff
@@ -82,6 +82,17 @@ volumeMounts: []
 # -- Headlamp pods volumes
 volumes: []
 
+persitentVolumeClaim:
+  # -- Enable Persistent Volume Claim
+  enabled: false
+  # -- Annotations to add to the persistent volume claim (if enabled)
+  annotations:
+    {}
+  # -- AccessModes for the persistent volume claim, eg: ReadWriteOnce, ReadOnlyMany, ReadWriteMany etc.
+  accessModes: []
+  # -- Size of the persistent volume claim, eg: 10Gi. Required if enabled is true.
+  size: ""
+
```

Comment:
As suggested above. Adding option for `storageClassName` and `selector` would be nice for the user.